packages/ws/src/strategies/sharding/WorkerShardingStrategy.ts

```diff
@@ -179,6 +173,61 @@ export class WorkerShardingStrategy implements IShardingStrategy {
 		worker.postMessage(payload);
 	}
 
+	private async setupWorker(workerData: WorkerData) {
+		const worker = new Worker(this.resolveWorkerPath(), { workerData });
+
+		await once(worker, 'online');
+		// We do this in case the user has any potentially long running code in their worker
+		await this.waitForWorkerReady(worker);
+
+		worker
+			.on('error', (err) => {
+				throw err;
+			})
+			.on('messageerror', (err) => {
+				throw err;
+			})
+			.on('message', async (payload: WorkerRecievePayload) => this.onMessage(worker, payload));
+
+		this.#workers.push(worker);
+		for (const shardId of workerData.shardIds) {
+			this.#workerByShardId.set(shardId, worker);
+		}
+	}
+
+	private resolveWorkerPath(): string {
+		if (!this.options.workerPath) {
```

Comment:
Instead of accessing the property over and over, we can define it as a variable.